app/controllers/v0/facilities/health_controller.rb

```diff
@@ -23,12 +23,7 @@ def show
   private
 
   def validate_params
-    begin
-      raise ArgumentError unless params[:bbox].length == 4
-      params[:bbox].each { |x| Float(x) }
-    rescue ArgumentError
-      raise Common::Exceptions::InvalidFieldValue.new('bbox', params[:bbox])
-    end
+    validate_bbox
```

Comment:
since this is part of the overridden method, would it be better to call super and make it more clear that you are overriding validate_params defined in FacilitiesController?